src/OpenTelemetry.AutoInstrumentation/Configuration/ConfigurationKeys.cs

```diff
@@ -84,6 +84,21 @@ public class ConfigurationKeys
     /// </summary>
     public const string Http2UnencryptedSupportEnabled = "OTEL_DOTNET_AUTO_HTTP2UNENCRYPTEDSUPPORT_ENABLED";
 
+    /// <summary>
+    /// Configuration key for setting the directory for the profiler's log files.
+    /// If set, this setting takes precedence over environment variable OTEL_DOTNET_AUTO_LOG_PATH.
+    /// If not set, default is
+    /// "%ProgramData%"\OpenTelemetry .NET AutoInstrumentation\logs\" on Windows or
+    /// "/var/log/opentelemetry/dotnet/" on Linux.
+    /// </summary>
+    public const string LogDirectory = "OTEL_DOTNET_AUTO_LOG_DIRECTORY";
+
+    /// <summary>
+    /// Configuration key for setting the path for the profiler's native log file.
+    /// Environment variable OTEL_DOTNET_AUTO_LOG_DIRECTORY takes precedence over this setting, if set.
+    /// </summary>
+    public const string LogPath = "OTEL_DOTNET_AUTO_LOG_PATH";
+
```

Comment:
Do we need to have both settings?

Reply:
Yeah, better to have a single env var and use the same for native and managed.

Another thing that we may want to consider post-beta is: the native log is one file for multiple applications so it is not symmetrical to the managed one. Perhaps it will make the user's life simpler if the logs of a single run were easier to "group".